<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="fileLoadButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>240</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add scans to stitch&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+O</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sliceSpinBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>430</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Viewed slice number&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
   </widget>
   <widget class="QLabel" name="sliceSpinLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>410</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Viewed slice:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="slicePlaneBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>430</y>
      <width>79</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Sagittal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Coronal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Transverse</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Diagonal 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Diagonal 2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>410</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Slice plane:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="scansList">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>181</width>
      <height>221</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::InternalMove</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="removeScanButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove scans to stitch&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>11</y>
      <width>351</width>
      <height>541</height>
     </rect>
    </property>
    <property name="transformationAnchor">
     <enum>QGraphicsView::AnchorUnderMouse</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="algorithmBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>360</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithm:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>590</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSaveSlice"/>
    <addaction name="actionExportSlice"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveSlice">
   <property name="text">
    <string>Save slice</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExportSlice">
   <property name="text">
    <string>Export slice</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
